dmoj/contrib/coci.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+from dmoj.contrib.testlib import ContribModule as TestlibContribModule
+from dmoj.error import InternalError
+from dmoj.result import CheckerResult
+
+
+class ContribModule(TestlibContribModule):
+    PARTIAL = 7

```

Comment:
This is inherited from `TestlibContribModule`, right? I don't think you need to declare it again.

Reply:
I have now removed `PARTIAL` field.